src/components/statevector_component/statevector.sh

```diff
@@ -30,7 +33,7 @@ create_statevector() {
     chmod 755 make_state_vector_file.py
 
     printf "\nCalling make_state_vector_file.py\n"
-    python make_state_vector_file.py $LandCoverFile $StateVectorFName $LatMin $LatMax $LonMin $LonMax $BufferDeg $LandThreshold $nBufferClusters
+    python make_state_vector_file.py $LandCoverFile $HemcoDiagFile $StateVectorFName $LatMin $LatMax $LonMin $LonMax $BufferDeg $LandThreshold $OffshoreEmisThreshold $nBufferClusters
```

Comment:
The parameters for this python script are getting crowded. Maybe we should just have it take in the config file path as a variable and have the script read the yaml file directly? Just a thought.